includes/commands/class-mu-migration-base.php

```diff
@@ -8,14 +11,14 @@
 	/**
 	 * Holds the command arguments
 	 *
-	 * @var Array
+	 * @var array
```

Comment:
`@var` fields can also use the short format:
```
/**
 * @var array Holds the command arguments
 */
```

Shall I change to this instead of the multiline one?
This is a taste thing, some like it, others don't.
I prefer the short version, to have it like the other docblocks.